backstop/tests/checkbox.html

```diff
@@ -1159,8 +1159,7 @@ <h3>Checkbox on left</h3>
           width: fit-content;
           display: flex;
           flex-direction: column;
-          align-items: flex-start;
-        "
+          align-items: flex-start;"
```

Comment:
double quotes in new line looked better in this situation..
If you put it all in new line and then press save, will be same formatting?

Reply:
When I ran lint, it removed leading whitespace like this:

```diff
-         "
+ "
```